packages/terra-table/src/SelectableTableRows.jsx

```diff
@@ -49,9 +64,35 @@ class SelectableTableRows extends React.Component {
     const initialOnClick = row.props.onClick;
 
     return (event) => {
-      // The default isSelectable attribute is either undefined or true, unless the consumer specifies the row's isSelectable attribute as false.
       if (row.props.isSelectable !== false) {
         this.handleOnChange(event, index);
+
+        const { liveRegion } = this.props;
```

Comment:
Should we go ahead and destructure all the props used in this function?

Reply:
https://github.com/cerner/terra-core/pull/2331/commits/3fb9b80ad6657aec97c4c26e072c7f421c0e62fc